packages/syntax-highlighter/index.js

```diff
@@ -11,3 +11,5 @@ module.exports = (code, lang, opts = { dark: false, tokenizeVariables: false })
   );
 
 module.exports.uppercase = require('./uppercase');
+
+module.exports.modes = modes;
```

Comment:
Have you tried importing this into readme yet? I feel like you may have problems because our regular webpack config doesnt have .jsx support. I would maybe add it as a separate modes.js file, then we can require it as `require('@readme/syntax-highlighter/modes')` without needing to touch jsx files.